DOCSP-33172 added meta description for top 250 pages in docs-node (#832)
mongodb/docs-node: source/usage-examples.txt

   :values: reference

.. meta:: 

>    :description: Learn how to set up sample data in Atlas database deployments for the Node.js driver usage examples.

Reviewer: Suggestion:

I think it could be good to mention MongoDB, and perhaps to separate the tasks to clarify the section contains both informational items.

```suggestion
   :description: Learn how to load sample data into a MongoDB Atlas deployment and run Node.js driver usage examples.
```
